core/src/main/java/org/elasticsearch/index/IndexingSlowLog.java

```diff
@@ -193,7 +193,9 @@ public String toString() {
                 String source = XContentHelper.convertToJson(doc.source(), reformat, doc.getXContentType());
                 sb.append(", source[").append(Strings.cleanTruncate(source, maxSourceCharsToLog)).append("]");
             } catch (IOException e) {
-                sb.append(", source[_failed_to_convert_]");
+                sb.append(", source[_failed_to_convert_[").append(e.getMessage()).append("]]");
+                sb.append(", reformat[").append(reformat).append("]");
```

Comment:
Looking at the issue this is trying to fix (#21946), I'm not sure if we need to include the "reformat" and the "x_content_type" in the slow log output.

Reply:
I pushed 96b88d46cf59a08c9b69e8433364d0c5abc04d30 . Thanks very much! 👍